synapse/config/emailconfig.py

```diff
@@ -74,19 +74,37 @@ def read_config(self, config, **kwargs):
             "renew_at"
         )
 
-        email_trust_identity_server_for_password_resets = email_config.get(
-            "trust_identity_server_for_password_resets", False
+        self.email_threepid_behaviour = (
+            # Have Synapse handle the email sending if account_threepid_delegate
+            # is not defined
+            "remote"
```

Comment:
I know this is not your code, but I think for sanity it would be good to define an [`enum`](https://docs.python.org/3/library/enum.html) for this